.drone.yml

```
@@ -440,7 +440,7 @@ steps:
   image: buildkite/puppeteer
   commands:
   - yarn wait-on http://$HOST:$PORT
-  - yarn -s test:accessibility --json > pa11y-ci-results.json
+  - yarn run --silent test:accessibility --json > pa11y-ci-results.json
```

```suggestion
  - yarn run test:accessibility --json > pa11y-ci-results.json
```
The `--silent` flag doesn't do anything
https://github.com/yarnpkg/berry/blob/67e046fec96402637e367b39502e185c82696db5/packages/plugin-essentials/sources/commands/run.ts#L55-L60

---

Oh nice, thanks